### ISSUE TITLE
Separate out filter options a bit?

### ISSUE THREAD

**Issue by mxmilkiib** [EskiMojo14/keycaplendar#50]
I.e. into plastic, profile, etc.

**Comment by EskiMojo14** [comment]
Adding more options for filtering/distinguishing sets by material and manufacturer is definitely something I've [considered](https://trello.com/c/7QCIj54T/75-add-way-to-distinguish-manufacturer), but I'm not sure what the best approach would be. 

Tracking profile, material, _and_ manufacturer on a set by set basis, as [keycapsets.com](https://keycapsets.com/) does, would require a large amount of work to go back and add the information to each existing set. On the other hand, having a separate database of profiles and corresponding material and manufacturer would definitely complicate matters.

It's still something I'll keep in mind, but it's pretty low priority in terms of things to add.

**Comment by mxmilkiib** [comment]
I'm referring to the existing data; given the 'Profile' filter section contains various "tags" that include ones solely for material types, others for profile type, and then for manufacturers. Could these not be separated visually to make it easier for the user?

**Comment by EskiMojo14** [comment]
The current Profile field is entirely based on common usage - Cherry profile keycaps are referred to by their manufacturer, other profiles are referred to by name, and some profiles are available in both materials and thus are separated as such. It's all based on the natural way people refer to keycaps.

**Comment by mxmilkiib** [comment]
I agree, but you just naturally distinguished between three types yourself, ergo not "out there" for there being worth in slightly separating the list out; why not have all "only shape" filter options first, "only material" second, "combined material/shape" third, "manufacturer" fourth, or so?

I think I do understand though now that you are noting before that all the existing profile options are lumped in the one taxonomy set, so extra data entry (or advanced scraping) would be needed to separate those out in the DB.

But even if they are all in the same set behind the scenes, could the placement in the filter sidebar not be more distinguished?

**Comment by EskiMojo14** [comment]
Separating them in the filter sidebar would be more confusing, in my personal opinion - I think it would imply that a set can be multiple things.

There aren't any profile names that are _solely_ material based. The material is currently only added in the rare case where the same profile is available from the same manufacturer/vendor in both materials. 

I think it could be definitely useful for people who aren't familiar with all the profiles to have material and manufacturers noted somehow, but I don't think separating them in the sidebar is the right solution.

**Comment by mxmilkiib** [comment]
Sure thing. Thanks for the feedback and consideration.
